docs/layers/s2-layer.md

```diff
@@ -74,7 +74,6 @@ new S2Layer({});
 To use pre-bundled scripts:
 
 ```html
-<script src="https://bundle.run/s2-geometry@1.2.10"></script>
 <script src="https://unpkg.com/deck.gl@^8.0.0/dist.min.js"></script>
```

Comment:
shouldn't this be updated to 8.1.0?

8.0.0 still requires s2-geometry in the html